pages/index.tsx

```diff
@@ -216,7 +162,7 @@ function GiftList({
 
   return (
     <div>
-      {giftData.length > 0 && (
+      {giftData && giftData.length > 0 && (
```

Comment:
Lisäsin tähän pienen tarkistuksen, joka tarkistaa onko `giftData` olemassa. Nopealla debauggauksella näyttäisi siltä, että `giftData` on ainoastaan alkurenderöinnissä `undefined`. Onko tuo ok vai olisiko hyvä tehdä jokin muokkaus?

Reply:
On oikein hyvä näin.